src/main/java/seedu/address/logic/ProfBookLogicManager.java

```diff
@@ -27,14 +30,16 @@ public class ProfBookLogicManager {
     private final Logger logger = LogsCenter.getLogger(LogicManager.class);
 
     private final State state;
-    //private final Storage storage = null;
+
+    private final ProfBookStorage storage;
     private final ProfBookParser profBookParser;
 
     /**
      * Constructs a {@code LogicManager} with the given {@code Model} and {@code Storage}.
      */
-    public ProfBookLogicManager(State state) {
+    public ProfBookLogicManager(State state, ProfBookStorage storage) {
         //todo : storage;
```

Comment:
If the todo is done, Should we remove it?

Reply:
Removed